db/migrate/20151223070312_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.string :commenter
```

Comment:
Yes, it'd be. But it all depends on the project requirements. So far our client would prefer to allow anyone to comment.

The restricted area is only for "content creators".

If any time in the future, the client comes and say "I need to block comments only for authenticated users", I do not have any problem in doing that and still keeping the old comments author's name